bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/handler/RenaultHandler.java

```diff
@@ -320,10 +336,39 @@ private void updateBattery(MyRenaultHttpSession httpSession) {
                     updateState(CHANNEL_CHARGING_REMAINING_TIME,
                             new QuantityType<Time>(chargingRemainingTime.doubleValue(), MINUTE));
                 }
+                ZonedDateTime batteryStatusUpdated = car.getBatteryStatusUpdated();
+                if (batteryStatusUpdated != null) {
+                    updateState(CHANNEL_BATTERY_STATUS_UPDATED, new DateTimeType(batteryStatusUpdated));
+                }
             } catch (RenaultNotImplementedException e) {
+                logger.warn("Disable cockpit battery update.");
                 car.setDisableBattery(true);
             } catch (RenaultForbiddenException | RenaultUpdateException e) {
+                logger.warn("Error updating battery status.", e);
             }
         }
     }
+
+    private void updateLockStatus(MyRenaultHttpSession httpSession) {
+        if (!car.isDisableLockStatus()) {
+            try {
+                httpSession.getLockStatus(car);
+                updateState(CHANNEL_LOCK_STATUS, new StringType(car.getLockStatus().name()));
+            } catch (RenaultNotImplementedException e) {
+                updateState(CHANNEL_LOCK_STATUS, new StringType(LockStatus.UNKNOWN.name()));
+                logger.warn("Disable lock status update.");
+                car.setDisableLockStatus(true);
+            } catch (RenaultForbiddenException | RenaultUpdateException e) {
+                logger.warn("Error updating lock status.", e);
```

Comment:
Under which circumstances will this happen? Will it be temporary failures (e.g. network issues) or could it be permanently rejection of such requests? In this case the warnings would keep reappearing in logs?

Reply:
Once the service is disabled due to a HTTP 404. 501, 502 the lock status service is disabled after the first failing call so that it does not keep reappearing in logs.

Reply:
I assume these three HTTP codes will cause `RenaultNotImplementedException `then? So are you saying that `RenaultForbiddenException` and `RenaultUpdateException` cannot happen unless `RenaultNotImplementedException` already was thrown prior to that, or that these exceptions will only be temporary?

Reply:
I am saying that RenaultForbiddenException and RenaultUpdateException cannot happen **IF** RenaultNotImplementedException was already was thrown prior to that.  

RenaultUpdateException etc can be temporary but the API is a bit unintuitive and it is hard to test...

However the binding is working well on my computer at the moment.

Reply:
Okay, let's see how this will work out in real life. I was concerned that these two exceptions will **not** trigger  `setDisableLockStatus`, so there could be a risk of repeating warnings in log.